### PR TITLE
Change CDR to DCR

### PR DESCRIPTION
**Pull request by lfarrell**
https://jira.lib.unc.edu/browse/HYC-732
https://jira.lib.unc.edu/browse/BXC-2368

### CODE REVIEW

File: admin/src/main/webapp/WEB-INF/jsp/common/header.jsp [hunk 1]
```diff
@@ -61,7 +61,7 @@
 	</ul>
 	<div class="dark shadowbottom">
 		<div class="cdr-header">
-			<h1 id="cdr-logo"><a href="${pageContext.request.contextPath}/" id="titlelink"><span class="dark-title">CAROLINA</span> <span class="light-title">DIGITAL</span> <span class="dark-title">REPOSITORY</span><span class="sub-title">Administration</span></a></h1>
+			<h1 id="cdr-logo"><a href="${pageContext.request.contextPath}/" id="titlelink"><span class="dark-title">DIGITAL</span> <span class="light-title"></span>Collections<span class="dark-title">REPOSITORY</span><span class="sub-title">Administration</span></a></h1>
```
Comment (bbpennel) [inline admin/src/main/webapp/WEB-INF/jsp/common/header.jsp]: Mixed case here

File: static/css/cdr-homepage.css [hunk 2]
```diff
@@ -1,262 +1,209 @@
 @import url("https://fonts.googleapis.com/css?family=Libre+Baskerville:400i|Open+Sans:400,600");
-/* line 17, ../sass/dcr.scss */
-* {
-    font-family: 'Open Sans', sans-serif;
-}
+body {
```
Comment (bbpennel) [inline static/css/cdr-homepage.css]: Why are all these formatting changes suddenly coming through?

File: static/service-unavailable.html [hunk 3]
```diff
@@ -31,7 +31,7 @@
             <div class="fluid-cap-left-wrap darkest">
               <div class="twocol fluid-cap-left">
                 <div class="cdr-header">
-                  <h1 id="cdr-logo"><a href="/" id="titlelink"><span class="white-title">CAROLINA</span> <span class="blue-title">DIGITAL</span> <span class="white-title">REPOSITORY</span></a></h1>
+                  <h1 id="cdr-logo"><a href="/" id="titlelink"><span class="white-title">DIGITAL</span> <span class="blue-title">Collections</span> <span class="white-title">REPOSITORY</span></a></h1>
```
Comment (bbpennel) [inline static/service-unavailable.html]: I think this needs to be updated with the new banner

File: static/css/cdr-homepage.css (hunk 2 above)
Comment (lfarrell) [reply static/css/cdr-homepage.css]: I added in the font and style updates in this pull request https://jira.lib.unc.edu/browse/BXC-2368

File: static/js/admin/src/IngestFromSourceForm.js [hunk 4]
```diff
@@ -228,7 +228,7 @@ define('IngestFromSourceForm', [ 'jquery', 'AbstractFileUploadForm', 'ModalLoadi
 					return false;
 				}
 				
-				var packagingType = (candidate.packagingType !== undefined) ? candidate.packagingType : 'http://cdr.unc.edu/DirectoryIngest';
+				var packagingType = (candidate.packagingType !== undefined) ? candidate.packagingType : 'http://dcr.unc.edu/DirectoryIngest';
```
Comment (bbpennel) [inline static/js/admin/src/IngestFromSourceForm.js]: Need to undo this

File: static/templates/admin/addFileForm.html [hunk 5]
```diff
@@ -14,7 +14,7 @@ <h3>Add File</h3>
     <div class="update_field">
         <input type="submit" id="object_submit_button" class="update_button" value="Ingest" />
     </div>
-    <input type="hidden" name="type" value="http://cdr.unc.edu/model/Simple" />
+    <input type="hidden" name="type" value="http://dcr.unc.edu/model/Simple" />
```
Comment (bbpennel) [inline static/templates/admin/addFileForm.html]: undo

File: static/templates/admin/ingestPackageForm.html [hunk 6]
```diff
@@ -10,7 +10,7 @@ <h3>Package</h3>
 	<div class="form_field">
 		<label>Package Type*</label>
 		<select name="type">
-			<option value="http://cdr.unc.edu/METS/profiles/Simple">CDR METS file</option>
+			<option value="http://dcr.unc.edu/METS/profiles/Simple">DCR METS file</option>
```
Comment (bbpennel) [inline static/templates/admin/ingestPackageForm.html]: undo

**Pull request merged**